packages/block-library/src/spacer/edit.native.js

```diff
@@ -0,0 +1,62 @@
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { View } from 'react-native';
+/**
+ * WordPress dependencies
+ */
+import {
+	PanelBody,
+	BottomSheet,
+} from '@wordpress/components';
+import { useState } from '@wordpress/element';
+import {
+	InspectorControls,
+} from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import styles from './editor.scss';
+
+const minSpacerHeight = 20;
+const maxSpacerHeight = 500;
+
+const SpacerEdit = ( { isSelected } ) => {
+	const [ spacerHeight, setSpacerHeight ] = useState( 20 );
+
+	const changeSpacerHeight = ( value ) => {
+		if ( value < minSpacerHeight ) {
+			return setSpacerHeight( minSpacerHeight );
+		} else if ( value > maxSpacerHeight ) {
+			return setSpacerHeight( maxSpacerHeight );
+		} return setSpacerHeight( value );
+	};
+
+	const getInspectorControls = () => (
```

Comment:
This can be either inlined into the `View` or assigned directly to the const:
```suggestion
	const inspectorControls = (
```

Reply:
I would like to keep it outside a `View`,  because on the next iteration once we will add a draggable functionality in can mess a code a bit. Assigning it directly to the const makes sense, thanks!

Reply:
I've decided to move it into `View` finally.